client/startup/startup.js

```diff
@@ -17,7 +17,7 @@ if (window.DISABLE_ANIMATION) {
 Meteor.startup(function() {
 	TimeSync.loggingEnabled = false;
 
-	UserPresence.awayTime = 300000;
+	UserPresence.awayTime = RocketChat.settings.get('Default_away_time')||300000;
```

Comment:
Thanks for making the changes, however this is not reactive and how you are making it reactive below in the `lib/settings.js` file is not correct. Passing a function to `RocketChat.settings.get('key', () => {})` will cause the function to be called whenever the setting's value changes. Please use that method instead of how are you doing it below. Thanks!

Reply:
could you may reevaluate, my changes?